pyasic/data/pools.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class PoolMetrics:
+    """A dataclass to standardize pool metrics returned from miners.
+    Attributes:
+
+    accepted: Number of accepted shares.
+    rejected: Number of rejected shares.
+    get_failures: Number of failures in obtaining work from the pool.
+    remote_failures: Number of failures communicating with the pool server.
+    stratum_active: Indicates if the miner is connected to the stratum server.
+    pool_rejected_percent: Percentage of rejected shares by the pool.
+    pool_stale_percent: Percentage of stale shares by the pool.
+    """
+
+    accepted: int
+    rejected: int
+    get_failures: int
+    remove_failures: int
+    stratum_active: bool
+
```

Comment:
The properties need a `field` here to properly include it into the dataclass.  See these - 
https://github.com/UpstreamData/pyasic/blob/c26b78aa014666ed2cf60123dbaf254b5719a23e/pyasic/data/__init__.py#L104-L106
https://github.com/UpstreamData/pyasic/blob/c26b78aa014666ed2cf60123dbaf254b5719a23e/pyasic/data/__init__.py#L266-L302